packages/quantification/src/__tests__/quantification.test.ts

```diff
@@ -10,6 +10,8 @@ import {
   MULTIPOLYGON_OUTPUT,
 } from './example-output';
 
+const MAX_NUMBER_OF_GRANDFATHERABLE_YEARS = 5;
```

Comment:
this is the number that is currently used in the expected test outputs... should probably write a test that checks that a different input leads to a different output